public/js/cin.js

```diff
@@ -0,0 +1,527 @@
+/**
+ * University Radio York Candidate Interview Night
+ */
+
+import { h, render } from "https://unpkg.com/preact@latest?module";
+import {
+  useState,
+  useEffect,
+  useRef,
+} from "https://unpkg.com/preact@latest/hooks/dist/hooks.module.js?module";
+import htm from "https://unpkg.com/htm?module";
+
+// Initialize htm with Preact
+const html = htm.bind(h);
+
+var api = "";
+var interviews = [];
```

Comment:
Surely this should be a state variable of some component, not a global?

Reply:
i mean, it could be either. i did it this way at the time cause it made debugging easier.